qa/cephfs/overrides/prefetch_dirfrags.yaml

```diff
@@ -0,0 +1,4 @@
+overrides:
+  ceph:
+    cephfs:
+      mds_dir_prefetch: true 
```

Comment:
Better to have the enable/disable as two yamls (explicitly setting the config to true/false in case we change the default someday).

Reply:
> Better to have the enable/disable as two yamls (explicitly setting the config to true/false in case we change the default someday).

done